cmd/buck/cli/archive.go

```diff
@@ -4,9 +4,11 @@ import (
 	"bytes"
```

Comment:
Here we're doing mostly what was described in the PR description about more checks and prompts to avoid common pitfalls.